packages/kit/src/exports/vite/preview/index.js

```diff
@@ -36,6 +36,10 @@ export async function preview(vite, vite_config, svelte_config) {
 
 	const dir = join(svelte_config.kit.outDir, 'output/server');
 
+	if (!existsSync(dir)) {
+		throw new Error(`Server files not found at ${dir}, did you run \`build\` first?`);
```

Comment:
```suggestion
		throw new Error(`Server files not found at ${dir}, did you run \`vite build\` first?`);
```

Reply:
I had that in my first iteration but removed the vite part because people usually run the build script in their package.json instead of invoking the vite CLI by itself.